pkg/middleware/csrf/csrf_test.go

```diff
@@ -180,7 +180,8 @@ func TestCSRF_Check(t *testing.T) {
 				require.NoError(t, err)
 			} else {
 				require.Error(t, err)
-				assert.EqualValues(t, tc.expectedStatus, err.(*errorWithStatus).HTTPStatus)
+				actual := err.(*errorWithStatus).HTTPStatus
+				assert.EqualValues(t, tc.expectedStatus, actual)
```

Comment:
Here we can either ignore the linter or do something like this:

```suggestion
				var actual *errorWithStatus
				require.True(errors.As(err, &actual))
				assert.EqualValues(t, tc.expectedStatus, actual.HTTPStatus)
```

Reply:
I see,